.eslintrc.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
     'extends': 'airbnb',
+    'rules': {
+      'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
Do we want to allow files with both these extensions to have JSX? Is this for tests?

Also, could you use "warn" instead of "1"?